common/utils/src/main/scala/org/apache/spark/util/SparkFileUtils.scala

```diff
@@ -197,6 +198,22 @@ private[spark] trait SparkFileUtils extends Logging {
       Files.isSameFile(path1, path2) || Files.mismatch(path1, path2) == -1L
     }
   }
+
+  def touch(file: File): Unit = {
+    if (file == null) {
+      throw new IllegalArgumentException(s"Invalid input file $file")
```

Comment:
Can we use a constant string for the error message here? Because the `file` is always null.

Reply:
Sure.